.github/workflows/draft-pdf.yml

```diff
@@ -0,0 +1,26 @@
+name: Open Journals PDF Generator
+uses: openjournals/openjournals-draft-action@v.1.0
+
+on: [push]
+
+jobs:
+  paper:
+    runs-on: ubuntu-latest
+    name: Paper Draft
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build draft PDF
+        uses: openjournals/openjournals-draft-action@master
+        with:
+          journal: joss
+          # This should be the path to the paper within your repo.
+          paper-path: paper.md
```

Comment:
Shouldn't this be` paper/paper.md`?

Reply:
I've created a branch for myself and made this change. I also wondered if this action is only triggered if a change is made to `paper/ ` but it's still not running. I'll just code up my own action and see what it does